containers/Connector/Connector.tsx

```diff
@@ -65,22 +66,29 @@ const useConnector = () => {
 
 	useEffect(() => {
 		const init = async () => {
-			// TODO: need to verify we support the network
 			const networkId = await getDefaultNetworkId();
 
-			const provider = loadProvider({
+			const _provider = loadProvider({
 				networkId,
 				infuraId: process.env.NEXT_PUBLIC_INFURA_PROJECT_ID,
 				provider: window.ethereum,
 			});
 			const useOvm = getIsOVM(networkId);
 
-			const snxjs = synthetix({ provider, networkId, useOvm });
+			const supportedNetworks = await snx.networks;
```

Comment:
If you use `@synthetixio/contracts-interface` you should be able to get a list of networks from: 

https://github.com/Synthetixio/js-monorepo/blob/4e76071b7b63ad112281899c2224b78cc46ff41d/packages/contracts-interface/src/types.ts#L5-L14

Only one that's missing is `local`, but we can add that in the future.